docs/source/pythonapi/examples/mgxs-part-iii.ipynb

```diff
@@ -1078,22 +1086,30 @@
       "\tDomain ID      =\t10000\n",
       "\tNuclide        =\tU235\n",
       "\tCross Sections [cm^-1]:\n",
-      "            Group 1 [0.625      - 20000000.0eV]:\t8.05e-03 +/- 3.37e-01%\n",
-      "            Group 2 [0.0        - 0.625     eV]:\t3.62e-01 +/- 6.02e-01%\n",
+      "            Group 1 [0.625      - 20000000.0eV]:\t8.05e-03 +/- 3.37e-01\n",
```

Comment:
So am I correct in understanding that the `print_xs` method is printing the std. dev. rather than the percent relative error? Or that we were printing the std. dev. out all along but reporting it as a percent, and are no longer doing so? IMHO the percent rel. err. is more relevant and easier to relate to than the std. dev. - what are your thoughts?